app/src/androidTest/java/com/duckduckgo/app/cta/CtaDaxHelperTest.kt

```diff
@@ -0,0 +1,188 @@
+        whenever(mockOnboardingStore.onboardingDialogJourney).thenReturn(null)
+        whenever(mockAppInstallStore.installTimestamp).thenReturn(System.currentTimeMillis() - TimeUnit.DAYS.toMillis(4))
+
+        val value = testee.addCtaToHistory("test")
+        assertEquals("test:3", value)
+    }
+
+    @Test
+    fun whenAddCtaToHistoryContainsHistoryThenConcatenateNewValue() {
+        val ctaHistory = "s:0-t:1"
+        whenever(mockOnboardingStore.onboardingDialogJourney).thenReturn(ctaHistory)
+        whenever(mockAppInstallStore.installTimestamp).thenReturn(System.currentTimeMillis() - TimeUnit.DAYS.toMillis(1))
+
+        val value = testee.addCtaToHistory("test")
+        val expectedValue = "$ctaHistory-test:1"
+
+        assertEquals(expectedValue, value)
+    }
+
+    @Test
+    fun whenCanSendPixelAndCtaNotPartOfHistoryThenReturnTrue() {
+        whenever(mockOnboardingStore.onboardingDialogJourney).thenReturn("s:0")
+        assertTrue(testee.canSendPixel(DaxBubbleCta.DaxEndCta(testee)))
+    }
+
+    @Test
+    fun whenCanSendPixelAndCtaIsPartOfHistoryThenReturnFalse() {
+        whenever(mockOnboardingStore.onboardingDialogJourney).thenReturn("e:0")
+        assertFalse(testee.canSendPixel(DaxBubbleCta.DaxEndCta(testee)))
+    }
+
+    @Test
+    fun whenCanSendPixelAndCtaIsHomePanelTypeThenReturnTrue() {
+        assertTrue(testee.canSendPixel(HomePanelCta.AddWidgetAuto))
+    }
+
+    @Test
+    fun whenGetNetworkPercentageForGoogleThenReturn90() {
+        assertEquals("90%", testee.getNetworkPercentage("Google"))
+    }
+
+    @Test
+    fun whenGetNetworkPercentageForFacebookThenReturn40() {
+        assertEquals("40%", testee.getNetworkPercentage("Facebook"))
+    }
+
+    @Test
+    fun whenIsFromSameNetworkDomainForFacebookThenReturnTrue() {
+        assertTrue(testee.isFromSameNetworkDomain("facebook"))
+    }
+
+    @Test
+    fun whenIsFromSameNetworkDomainForGoogleThenReturnTrue() {
+        assertTrue(testee.isFromSameNetworkDomain("google"))
+    }
+
+    @Test
+    fun whenIsFromSameNetworkDomainForAmazonThenReturnFalse() {
+        assertFalse(testee.isFromSameNetworkDomain("amazon"))
+    }
+
+    @Test
+    fun whenMoreThanTwoMajorTrackersBlockedReturnFirstTwoWithMultipleString() {
+        val trackers = listOf(
+            TrackingEvent("facebook.com", "facebook.com", blocked = true, entity = TestEntity("Facebook", "Facebook", 9.0), categories = null),
+            TrackingEvent("other.com", "other.com", blocked = true, entity = TestEntity("Other", "Other", 9.0), categories = null),
+            TrackingEvent("amazon.com", "amazon.com", blocked = true, entity = TestEntity("Amazon", "Amazon", 9.0), categories = null)
+        )
+
+        val value = testee.getTrackersBlockedCtaText(mockActivity, trackers)
+        assertEquals("<b>Facebook, Other</b>withMultiple", value)
+    }
+
+    @Test
+    fun whenTwoMajorTrackersBlockedReturnThemWithMultipleString() {
```

Comment:
Test name says `withMultipleString`, asserted value is `withZero`.

Reply:
Changed!